### ISSUE TITLE
When dismissing the terminal accessibility help by pressing the esc key, the focus is not moved back to the terminal.

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#173909]
Type: <b>Bug</b>
CC @isidorn 
CC @meganrogge 

I create a terminal in VSCode, press shift+alt+f1 to display the accessibility help, and then press the esc key to dismiss the displayed information. I believe that the focus should be moved back to the terminal but it doesn't. I need to press shift+tab to move the focus back to the terminal.
I tested using orca but I believe the same happens using nvda, I'll test later.

VS Code version: Code - Insiders 1.76.0-insider (853b32bad26ed5ffe2d29b95be45adc7f753e35a, 2023-02-09T05:23:03.109Z)
OS version: Linux x64 6.1.9-200.fc37.x86_64
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 2999)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|1, 1, 1|
|Memory (System)|15.27GB (8.03GB free)|
|Process Argv|-n --crash-reporter-id bc02adbd-6c3a-44b8-af92-55103a608af7|
|Screen Reader|yes|
|VM|0%|
|DESKTOP_SESSION|gnome-xorg|
|XDG_CURRENT_DESKTOP|GNOME|
|XDG_SESSION_DESKTOP|gnome-xorg|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (37)</summary>

Extension|Author (truncated)|Version
---|---|---
doxdocgen|csc|1.4.0
vscode-markdownlint|Dav|0.49.0
vscode-chatgpt|gen|3.5.9
rest-client|hum|0.25.1
better-cpp-syntax|jef|1.17.2
azure-pipelines|ms-|1.208.0
vscode-docker|ms-|1.23.3
csharp|ms-|1.25.4
vscode-kubernetes-tools|ms-|1.3.11
remote-containers|ms-|0.278.0
remote-ssh|ms-|0.97.2023020215
remote-ssh-edit|ms-|0.84.0
azure-account|ms-|0.11.3
cmake-tools|ms-|1.13.45
cpptools|ms-|1.14.1
cpptools-extension-pack|ms-|1.3.0
makefile-tools|ms-|0.6.0
powershell-preview|ms-|2023.2.0
remote-explorer|ms-|0.3.2023020609
vsliveshare|ms-|1.0.5828
fabric8-analytics|red|0.3.6
java|red|1.15.2023020903
vscode-xml|red|0.25.2023020103
vscode-yaml|red|1.11.0
sonarlint-vscode|Son|3.14.0
cmake|twx|0.0.17
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-boot-dev-pack|vmw|0.2.1
vscode-spring-boot|vmw|1.44.0
vscode-java-debug|vsc|0.48.0
vscode-java-dependency|vsc|0.21.2023013100
vscode-java-pack|vsc|0.25.2023020900
vscode-java-test|vsc|0.37.2023013002
vscode-maven|vsc|0.40.2023020103
vscode-spring-boot-dashboard|vsc|0.10.1
vscode-spring-initializr|vsc|0.11.2

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256198
vstes627cf:30244335
vslsvsres303:30308271
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30404738
cppdebug:30492333
vsclangdf:30492506
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30618038
pylantcb52:30590116
pyindex848:30611229
nodejswelcome1:30587009
pyind779:30611226
pythonsymbol12:30651887
6233i204:30659907

```

</details>
